### DISCUSSION TITLE
Possible to make notifications disappear when read in IM client?

### DISCUSSION THREAD

**Discussion by aereaux** [ErikReider/SwayNotificationCenter#185] [Q&A] [answered]
Hello,

I use gajim and dino on my desktop to chat with people on XMPP.  On platforms like android the notifications will go away when I read messages, but with SwayNC they until dismissed in SwayNC.  Would it be possible to make it so that step isn't needed?  If so, would it require modifications in SwayNC or my chat clients?

Thanks

**Comment by ErikReider** [comment, marked as answer]
That would be the clients job to call the deletion signal

**Reply by aereaux** [reply]
OK, thanks.  I was thinking it might be something like that.
